gymnasium_robotics/envs/fetch/pick_and_place.py

```diff
@@ -130,6 +130,7 @@ class MujocoFetchPickAndPlaceEnv(MujocoFetchEnv, EzPickle):
 
     ## Version History
 
+    * v4: Fixed bug: the initial state not matching the initial state description in the documentation. Fetch environments' initial states after reset now align with the documentation.(related [GitHub issue](https://github.com/Farama-Foundation/Gymnasium-Robotics/issues/251))
```

Comment:
* v4: Fixed bug where initial state did not match initial state description in documentation. Fetch environments' initial states after reset now match the documentation (related [GitHub issue](https://github.com/Farama-Foundation/Gymnasium-Robotics/issues/251)).